lib/rules/widget-not-inline-matches.js

```diff
@@ -15,14 +15,7 @@ const matchesFns = [
 ];
 
 function isWidgetType(vNode) {
-  const role = getRole(vNode);
-  if ('option' === role) {
-    return false;
-  } else if (['combobox', 'listbox'].includes(role)) {
-    return true;
-  }
-  const roleType = getRoleType(role);
-  return roleType === 'widget';
+  return getRoleType(vNode) === 'widget';
```

Comment:
So this will return `true` for the `option` role, is that OK?

Reply:
They do, but they're pulled out for not being focusable.